aospy/test/test_data_loader.py

```diff
@@ -369,5 +371,22 @@ def test_data_name_gfdl_seasonal(self):
         self.assertEqual(result, expected)
 
 
+class LoadVariableTestCase(unittest.TestCase):
```

Comment:
@spencerahill I think net the coverage decrease is because I removed a number of lines of code that would be run (but never actually did anything meaningful).  This is the test case that should definitively make sure this bug is fixed.